src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -344,7 +344,17 @@ private bool UpdateSettingsAndCreateCheckpoint(bool updateAttempted = true)
             {   // create temporary copy of settings.xml             
                 this.CreateTempCopyOfSettingsFile();
 
-                this._settingsManager.UpdateSettings(this._settingsManager.TempCopyofSettingsFilePath);
+                try
+                {
+                    this._settingsManager.UpdateSettings(this._settingsManager.TempCopyofSettingsFilePath);
+                    string message = "Attempt to update settings was successful.";
+                    this._nodeAgentSfUtility.ReportHealth(WUOperationSetting, message, HealthState.Ok, 1);
```

Comment:
>ReportHealth [](start = 45, length = 12)

You are posting this on the service. This kind of exception should be on the instance.

Reply:
The warning when LoadSettings in NodeAgentNTService is also posted on Service. Plus, we have used "WUOperationSetting"  to post these kind of events only.

Reply:
We are not posting anything on instance of NodeAgentService.